### DISCUSSION TITLE
Implement react-intl with SSG

### DISCUSSION THREAD

**Discussion by ystewart** [vercel/next.js#16188] [Help]
Hello,

I just implemented the update Apollo v3  in my NextJS project according the [with-apollo](https://github.com/vercel/next.js/tree/canary/examples/with-apollo) example provided in this repository. I also have react-intl implemented in the same project according to the [with-react-intl](https://github.com/vercel/next.js/tree/canary/examples/with-react-intl) example, but that implementation results in a purely server-side rendered application.  My question is has anyone implemented the react-intl in next without using getInitialProps in a custom _app.js in to be able to use SSG?

**Comment by sergioviniciuss** [comment]
it's quite sad no one replied in here so far. I'm also very interested in a solution for this.
